crates/pallet-domains/src/lib.rs

```diff
@@ -268,6 +333,14 @@ mod pallet {
         DomainRuntimeUpgraded {
             runtime_id: RuntimeId,
         },
+        OperatorRegistered {
+            operator_id: OperatorId,
+            domain_id: DomainId,
+        },
+        OperatorNominated {
+            operator_id: OperatorId,
+            nominator_id: T::AccountId,
```

Comment:
Should this be `NominatorId` for consistency?

Reply:
Ahh yes. Missed it somehow.